packages/orbit-components/src/primitives/MobileDialogPrimitive/__tests__/index.test.js

```diff
@@ -1,29 +1,37 @@
 // @flow
 import * as React from "react";
-import { mount } from "enzyme";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 import MobileDialog from "../index";
-import Button from "../../../Button";
 
-describe("MobileDialogPrimitive", () => {
-  const content = "My text link";
-  const tabIndex = "1";
-  const dataTest = "test";
+jest.mock("../../../utils/randomID", () => name => name);
```

Comment:
we have it already mocked in `enzymeConfig.js`, but since it supposed to be removed later, we will move it to common jest config or will mock it every time like that?

Reply:
If you're referring to this:

```js
const mockMath = Object.create(global.Math);
mockMath.random = () => 0.5;
global.Math = mockMath;
```

I don't think that's a good way because we might be later be using `Math.random` for something else, and might not necessarily want to mock it. I'm a little uncomfortable about mocking it globally like that, I think it's best to always mock `randomID` explicitly in tests where that's needed. Alternatively, we can move this `jest.mock` call to the setup file, I just want to avoid mocking `Math.random`.

Reply:
okay, i just decided to ask to be sure :) thanks